docs/docs/api-ref.md

```diff
@@ -5,257 +5,3010 @@ layout: docs
+Name | Description   
+:--- | :---   
+`username` | The name of the user whose credentials will be used to sign in.   
+`password` | The password of the user.     
+`site_id` | This corresponds to the `contentUrl` attribute in the Tableau REST API. The `site_id` is the portion of the URL that follows the `/site/` in the URL. For example, "MarketingTeam" is the `site_id` in the following URL *MyServer*/#/site/**MarketingTeam**/projects. To specify the default site on Tableau Server, you can use an empty string `''`  (single quotes, no space).  For Tableau Online, you must provide a value for the `site_id`.  
+`user_id_to_impersonate` |  Specifies the id (not the name) of the user to sign in as.   
+
+Source file: models/tableau_auth.py
+
+**Example**
 
 ```py
-Auth.sign_out()
+import tableauserverclient as TSC
+# create a new instance of a TableauAuth object for authentication
+
+tableau_auth = TSC.TableauAuth('USERNAME', 'PASSWORD', site_id='CONTENTURL')
+
+# create a server instance
+# pass the "tableau_auth" request object to server.auth.sign_in()
 ```
 
-## Sites
+<br>
+<br>  
+
+### Auth methods
+The Tableau Server Client provides two methods for interacting with authentication resources. These methods correspond to the sign in and sign out endpoints in the Tableau Server REST API.
 
-Source files: server/endpoint/sites_endpoint.py, models/site_item.py
 
-### Create Site
+Source file: server/endpoint/auth_endpoint.py
 
-Creates a new site for the given site item object.
+<br>
+<br> 
+
+#### auth.sign in
 
 ```py
-Sites.create(new_site_item)
+auth.sign_in(auth_req)
 ```
 
-Example:
+Signs you in to Tableau Server.
+
+
+The method signs into Tableau Server or Tableau Online and manages the authentication token. You call this method from the server object you create. For information about the server object, see [Server](#server). The authentication token keeps you signed in for 240 minutes, or until you call the `auth.sign_out` method. Before you use this method, you first need to create the sign-in request (`auth_req`) by creating an instance of the `TableauAuth`. To call this method, create a server object for your server. For more information, see [Sign in and Out](sign-in-out).
+
+REST API: [Sign In](http://onlinehelp.tableau.com/current/api/rest_api/en-us/help.htm#REST/rest_api_ref.htm#Sign_In%3FTocPath%3DAPI%2520Reference%7C_____77){:target="_blank"}
+
+**Parameters**
+
+`auth_req` : The `TableauAuth` object that holds the sign-in credentials for the site. 
+
+
+**Example**
 
 ```py
-new_site = TSC.SiteItem(name='Tableau', content_url='tableau', admin_mode=TSC.SiteItem.AdminMode.ContentAndUsers, user_quota=15, storage_quota=1000, disable_subscriptions=True)
-self.server.sites.create(new_site)
+import tableauserverclient as TSC
+
+# create a auth request
+tableau_auth = TSC.TableauAuth('USERNAME', 'PASSWORD')
+
+# create an instance for your server
+server = TSC.Server('http://SERVER_URL')
+
+# call the sign-in method with the request
```

Comment:
You call sign in with the object/resource/whatever-you-pick not the request, you're making a request